mops/include/mops/opsa.hpp

```diff
@@ -59,7 +59,35 @@ template <typename scalar_t>
 void MOPS_EXPORT outer_product_scatter_add(Tensor<scalar_t, 2> output,
                                            Tensor<scalar_t, 2> A,
                                            Tensor<scalar_t, 2> B,
+                                           Tensor<int32_t, 1> first_occurences,
```

Comment:
I would prefer not to change the API if we can. Is this something we can compute ourself inside the function instead?